egs/swbd/s5c/local/chain/run_tdnn_6z.sh

```diff
@@ -55,7 +55,7 @@ final_layer_normalize_target=0.5
 num_jobs_initial=3
```

Comment:
You can reuse the lattices and trees from the previous experiments. You should be able to start this experiment from stage 12.